src/utils/getVisibleTodos.ts

```diff
@@ -0,0 +1,12 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export function getVisibleTodos(todos: Todo[], filter: Filter) {
+  if (filter !== Filter.All) {
+    return [...todos].filter(todo => {
```

Comment:
```suggestion
    return todos.filter(todo => {
```
You don't need to create a copy as filter doesn't mutate an array